sdcm/nemesis.py

```diff
@@ -58,6 +59,20 @@ class UnsupportedNemesis(Exception):
     """ raised from within a nemesis execution to skip this nemesis"""
 
 
+class CompactionStrategy(Enum):
```

Comment:
lets find a better place for this class, it doesn't belong in nemesis

Reply:
let's create a new file called `compaction.py` inside `test_lib/`, and put this class and function related to compaction like `get_compaction_strategy()`

Reply:
agree

Reply:
done